tox.ini

```diff
@@ -19,6 +24,8 @@ setenv =
 deps =
 	django20: Django>=2.0,<2.1
 	django21: Django>=2.1,<2.2
+	django22: Django>=2.2,<3
+	django30: Django>=3.0a,<3.1
```

Comment:
3.0b1

Reply:
It's `>=` so it'll be whatever the latest version is.

Reply:
then 3.0a1 as it's on pypi

Reply:
It's not gonna change what's installed but sure, changed.